gcloud-java-examples/README.md

```diff
@@ -69,6 +69,23 @@ To run examples from your command line:
   $mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.DatastoreExample" -Dexec.args="my_name delete"
   ```
 
+  Here's an example run of `DnsExample`.
+  
+  Note that you have to enable the Google Cloud DNS API on the [Google Developers Console][developers-console] before running the following commands.
+  Note that the example creates and deletes dns records of type A only. Operations with other record types are not implemented in the example.
```

Comment:
Nit: capitalize "dns".

Also, can we add an "Also" before the second "Note that"?